deploy/kubernetes/overlays/lockrelease/configmap_rbac.yaml

```diff
@@ -71,6 +71,9 @@ subjects:
 - kind: ServiceAccount
   name: filestore-lockrelease-controller-sa
   namespace: gcp-filestore-csi-driver
+- kind: ServiceAccount
```

Comment:
It's best to create a new role binding for the lock release controller `filestorecsi-lockrelease-controller-role-binding` and not couple the two